tuf/api/metadata.py

```diff
@@ -504,16 +504,33 @@ class Root(Signed):
             supports consistent snapshots.
         keys: A dictionary that contains a public key store used to verify
             top level roles metadata signatures::
-            {
-                '<KEYID>': <Key instance>,
-                ...
-            },
```

Comment:
is this change of keyid content intentional? I think the opposite change was just done a week ago

Reply:
You are right, this is an unintentional change coming from the time when I had those changes before we added Key and Role classes.
Will fix it.
Great catch!